README.md

```diff
@@ -1,3 +1,7 @@
+[![npm](https://img.shields.io/npm/v/idb-pconnector?logo=npm)](https://www.npmjs.com/package/idb-pconnector)
+[![ryver](https://img.shields.io/badge/Ryver-Chat-blue)](https://ibmioss.ryver.com/index.html#forums/1000127)
```

Comment:
They only problem with this link is that Ryver gives no way to sign up if they don't already have an account. For that, they need the signup link: https://ibmioss.ryver.com/application/signup/members/9tJsXDG7_iSSi1Q